### ISSUE TITLE
history search popup do not compatible with long command line.

### ISSUE THREAD

**Issue by neil-acc** [warpdotdev/Warp#1038]
### Describe the bug

```text
when the history contains a long command line, the text in the popup is very strange.

**Refer to the screenshot**
```

### Screenshots

<img width="1028" alt="Xnip2022-04-14_02-13-59" src="https://user-images.githubusercontent.com/75743591/163244445-3293c5d8-bf86-4c16-bc6a-d062a802dc6e.png">

### Operating System

MacOS

### OS Version

12.3.1

### Additional context

```text
If the long command line supports the mouse hover to prompt full content will be a good experience :)
```

**Comment by ivenhov** [comment]
Agree, even when multiple search words are used it still may not be enough to narrow down results.
My example
![Screenshot 2022-05-04 at 09 46 26](https://user-images.githubusercontent.com/778457/166650072-bee23440-7ec6-468a-816f-c70a69989462.png)


I think history window should be resizable and Warp should remember adjusted size between sessions. 
I instinctively tried to resize it realising it is not possible.
In addition mouse hover over would be great as suggested, but not sufficient since it requires hovering over many commands to find correct one.

**Comment by elviskahoro** [comment]
Thanks for pointing this out! Will cicrcle back when it's in progress!

**Comment by albertfc** [comment]
Happens to me too. Maybe a good behaviour could be the same you have for command flags search, a second window to the right with the full command expanded
<img width="723" alt="Screenshot 2023-01-19 at 08 26 29" src="https://user-images.githubusercontent.com/1190695/213380986-86c377bc-2932-4e48-ac98-2dd3d48d6743.png">
Thanks!

**Comment by lujop** [comment]
Yes, it's very small. I completely agree with invenhov proposal.
Or at least it's width should be configurable from preferences
